src/analysis/FVA/fluxVariability.m

```diff
@@ -156,11 +151,7 @@
 LPproblem.S = LPproblem.A;%needed for sparse optimisation
 
 %solve to generate initial basis
-if ~isfield(model,'osense')
-    LPproblem.osense = -1;
-else
-    LPproblem.osense = model.osense; 
-end
+LPproblem.osense = -1;

```

Comment:
These lines have been added in PR #843 (rebase error). May you add them again? Thanks.